django/website/main/templates/base.html

```diff
@@ -23,20 +23,18 @@ <h1 class="logo"><a href="/">{% settings_value "SITE_NAME" %}</a></h1>
         {% block logframe_switching_form %}
         {% if request.user and request.user.is_authenticated %}
         <form id="logframes-switch" action="{% url "switch-logframes" %}" method="post">{% csrf_token %}
-            <select id="logframes-switch-select" name="logframe" onchange="this.form.submit();"{% if logframe_list|length == 1 %}disabled="disabled"{% endif %}>
+            <select id="logframes-switch-select" name="logframe" onchange="this.form.submit();">
             	{% for logframe in logframe_list %}<option value="{{ logframe.id }}"{% if request.user.preferences.last_viewed_logframe == logframe %} selected="selected"{% endif %}>{{ logframe.name }}</option>{% endfor %}
+            	<option value="-1">{% trans "+ Create a new logframe" %}</option>
 			</select>
 		</form>
 		{% endif %}
 		{% endblock logframe_switching_form %}
         {% if request.user and request.user.is_authenticated %}
-         {% if perms.logframe.add_logframe %}
-            <div class="add-logframe"><a href="{% url "create-logframe" %}">Create Logframe</a></div>
-         {% endif %}
-         <div class="account-links-auth"><a href="{% url "logout" %}">Log out</a></div>
          {% if perms.contacts.add_user %}
             <div class="navigation-link"><a href="{% url "contact_list" %}" id="nav-people">People</a></li></div>
          {% endif %}
+         <div class="account-links-auth"><a href="{% url "logout" %}">Log out</a></div>
          {% if request.user.preferences.last_viewed_logframe %}
             <div class="navigation-link"><a href="{% url "logframe-dashboard" request.user.preferences.last_viewed_logframe.slug %}" id="nav-dashboard">Dashboard</a></div>
```

Comment:
We now have a link called Dashboard and a page called Dashboard but one doesn't link to the other